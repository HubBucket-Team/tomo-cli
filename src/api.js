import * as utils from './utils';
import * as common from './utils/common';

export default {
    ...common,
    ...utils
};