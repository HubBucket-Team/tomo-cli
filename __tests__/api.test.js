import api from '../src/api';

describe('API', () => {
    test('exports utils', () => {
        expect(Object.keys(api)).toMatchSnapshot();
    });
});