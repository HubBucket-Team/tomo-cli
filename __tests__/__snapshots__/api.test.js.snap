// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`API exports utils 1`] = `
Array [
  "dict",
  "parse",
  "maybeApply",
  "getBinDirectory",
  "format",
  "someDoExist",
  "someDoExistSync",
  "allDoExist",
  "allDoExistSync",
  "allDoNotExist",
  "allDoNotExistSync",
  "populateQueue",
  "isUniqueTask",
  "isValidTask",
  "withOptions",
  "choose",
  "getIntendedInput",
  "getVersions",
  "install",
  "uninstall",
  "BabelConfigModuleEditor",
  "EslintConfigModuleEditor",
  "PackageJsonEditor",
  "BsConfigJsonEditor",
  "PostcssConfigEditor",
  "PurgecssConfigEditor",
  "RollupConfigEditor",
  "WebpackConfigEditor",
]
`;
